### PR TITLE
Set maximumLatency variable

### PR DESCRIPTION
**Pull request by Tal-or**
In the last update, setting the ```maximumLatency``` via the environment variables value was skipped (bug).
This update will set the values correctly.
Moreover, this update introduces a new unified environment variable for the maximum latency that will allow the user to use the same environment variable for all tests.
The specific environment variables will have precedence over this unified one.

Signed-off-by: Talor Itzhak <titzhak@redhat.com>

### CODE REVIEW

File: functests/4_latency/latency.go [hunk 1]
```diff
@@ -348,21 +353,30 @@ func extractLatencyValues(testName string, exp string, node *corev1.Node) string
...
 		testName != strings.ToUpper(cyclictestTestName) &&
 		testName != strings.ToUpper(hwlatdetectTestName) {
 		return fmt.Errorf("testName variable has incorrect value %q", testName)
 	}
 
+	var err error
+	unifiedMaxLatencyEnv := os.Getenv("MAXIMUM_LATENCY")
+	if unifiedMaxLatencyEnv != "" {
+		maximumLatency, err = strconv.Atoi(unifiedMaxLatencyEnv)
+		return fmt.Errorf("err: %v the environment variable MAXIMUM_LATENCY has incorrect value %q", err, unifiedMaxLatencyEnv)
```
Comment (fromanirh) [inline functests/4_latency/latency.go:372]: not sure this is correct. If we have the var set we parse as int... and then we always fail?

Comment (Tal-or) [reply functests/4_latency/latency.go]: Copied it with the original mistake.
Thanks!

File: functests/4_latency/latency.go [hunk 2]
```diff
@@ -348,22 +353,34 @@ func extractLatencyValues(testName string, exp string, node *corev1.Node) string
...
 	if testName != strings.ToUpper(oslatTestName) &&
 		testName != strings.ToUpper(cyclictestTestName) &&
 		testName != strings.ToUpper(hwlatdetectTestName) {
 		return fmt.Errorf("testName variable has incorrect value %q", testName)
 	}
 
+	var err error
+	unifiedMaxLatencyEnv := os.Getenv("MAXIMUM_LATENCY")
+	if unifiedMaxLatencyEnv != "" {
+		if maximumLatency, err = strconv.Atoi(unifiedMaxLatencyEnv); err != nil {
```
Comment (fromanirh) [inline functests/4_latency/latency.go:371]: in general (let's not change this just yet) we should not set non-local variables by side effect. A better way:
```
maximumLatency, err = strconv.Atoi(unifiedMaxLatencyEnv)
if err != nil {
    ...
}
```

**Review by fromanirh — COMMENTED** [review]
/lgtm
/hold
looks good, but how we can test this before the merge? is it covered by existing tests (which run automatically on CI)?

**Comment by Tal-or** [comment]
> /lgtm
> /hold
> looks good, but how we can test this before the merge? is it covered by existing tests (which run automatically on CI)?

This test should be consumed by the cnf-feature-deploy. I'm not aware to a test framework which test our tests

**Pull request merged**